code/datums/riding.dm

```diff
@@ -275,4 +275,58 @@
 		handle_vehicle_layer()
 		handle_vehicle_offsets()
 	else
-		user << "<span class='notice'>You'll need something  to guide the [ridden.name].</span>"
\ No newline at end of file
+		user << "<span class='notice'>You'll need something  to guide the [ridden.name].</span>"
+
+//CYBORGS. NO, THEY ARE NOT ANIMALS.
+/datum/riding/cyborg
+	keytype = null
+	vehicle_move_delay = 1
+
+/datum/riding/cyborg/proc/ride_check(mob/user)
+	if(user.incapacitated())
+		ridden.unbuckle_mob(user)
+		return
+
+/datum/riding/cyborg/handle_vehicle_layer()
+	if(ridden.dir == SOUTH)
+		ridden.layer = ABOVE_MOB_LAYER
+	else
+		ridden.layer = OBJ_LAYER
+	if(!ridden.buckled_mobs)
+		ridden.layer = MOB_LAYER
+
+/datum/riding/cyborg/handle_vehicle_offsets()
+	if(ridden.has_buckled_mobs())
+		for(var/mob/living/M in ridden.buckled_mobs)
+			if(!istype(M, /mob/living))
+				continue
+			M.setDir(ridden.dir)
+			switch(ridden.dir)
+				if(NORTH)
+					M.pixel_x = 0
+					M.pixel_y = 4
+				if(SOUTH)
+					M.pixel_x = 0
+					M.pixel_y = 4
+				if(EAST)
+					M.pixel_x = -6
+					M.pixel_y = 3
+				if(WEST)
+					M.pixel_x = 6
+					M.pixel_y = 3
+
+/datum/riding/cyborg/proc/on_vehicle_move()
+	for(var/mob/living/M in ridden.buckled_mobs)
+		ride_check(M)
```

Comment:
then do it in move() since this proc is already firing on move instead of snowflaking code you fuckshit